### PR TITLE
Update for RC6 chain and GSRPC versions

### PR DESCRIPTION
**Pull request by mikiquantum**

### CODE REVIEW

File: centchain/events.go [hunk 1]
```diff
@@ -148,13 +148,6 @@ type Events struct {
 	ChainBridge_RelayerAdded            []EventRelayerAdded                   //nolint:stylecheck,golint
 	ChainBridge_RelayerThresholdChanged []EventRelayerThresholdChanged        //nolint:stylecheck,golint
 	Nfts_DepositAsset                   []EventNFTDeposited                   //nolint:stylecheck,golint
-	Council_Proposed                    []types.EventCollectiveProposed       //nolint:stylecheck,golint
```
Comment (mikiquantum) [inline centchain/events.go:151]: Already part of GSRPC events

File: build/scripts/docker/docker-compose-cc.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   cc:
-    image: "centrifugeio/centrifuge-chain:20200625211147-5e0e5e1"
+    image: "centrifugeio/centrifuge-chain:20200928161532-a960a81"
```
Comment (mikiquantum) [inline build/scripts/docker/docker-compose-cc.yml:4]: RC6 image

**Review by vedhavyas — APPROVED** [review]

**Comment by vedhavyas** [comment]
@mikiquantum looks like there is another PR with substrate v2.0.0 changes https://github.com/centrifuge/go-substrate-rpc-client/pull/100. Is it worth the time to wait for that to get in and update this accordingly ?

**Comment by mikiquantum** [comment]
@vedhavyas I would not, since that version might not be compatible with our chain.

**Pull request merged**